src/test/java/de/adorsys/keycloak/config/service/ImportAuthenticationFlowsIT.java

```diff
@@ -472,10 +478,15 @@ AuthenticationFlowRepresentation assertThatBrowserFlowIsUpdated(int expectedNumb
         AuthenticationExecutionExportRepresentation myForms = getExecutionFlowFromFlow(flow, "my forms");
         assertThat(myForms, notNullValue());
         assertThat(myForms.getRequirement(), is("ALTERNATIVE"));
-        assertThat(myForms.getPriority(), is(3));
         assertThat(myForms.isUserSetupAllowed(), is(false));
         assertThat(myForms.isAutheticatorFlow(), is(true));
 
+        if (VersionUtil.ge(KEYCLOAK_VERSION, "25")) {
```

Comment:
Are there some changes regarding the order/priority in the flows in version 25?

Reply:
Explanation above 😉